src/Core/Style.js

```diff
@@ -155,16 +145,158 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
 }
 
 /**
- * A Style is an object that defines the visual appearance of {@link
+ * @description An object that can contain any properties
+ * (order, zoom, fill, stroke, point, text or/and icon)
+ * and sub properties of a Style.
+ * @typedef {Object} Style_Interface
```

Comment:
I think it should be called `StyleOptions` instead to be more consistent (e.g. with CameraTransformOptions)